sql-schema/109.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE tmp_table LIKE config;
+ALTER IGNORE TABLE tmp_table ADD UNIQUE INDEX uniqueindex_configname (config_name);
```

Comment:
No it's fine, especially if it's from the Laravel stuff as I had to fix some issues with our schema build to port it over to Laravel so much prefer if it's right from day one.